src/app/files/file-explorer.js

```diff
@@ -99,8 +100,9 @@ function fileExplorer (appAPI, files) {
         })) : undefined
       }
     },
-    formatSelf: function (key, data, li) {
-      var isRoot = data.path.indexOf('/') === -1
+    formatSelf: function formatSelf (key, data, li) {
+      var isRoot = !data.path.indexOf('browser') && data.path.length === 'browser'.length
```

Comment:
why `var isRoot = data.path.indexOf('/') === -1` is not more applicable? I think it still is

Reply:
hmm... maybe.
I think i had  scenario where this wasnt working, but don't remember what it was.

:/

Reply:
could you rollback for now?

Reply:
ok